### PR TITLE
Messaging Product Page Fixes

### PR DESCRIPTION
**Pull request by ItzNotABug**
## What does this PR do?

This PR addresses the following -
1. Open links on the same page
2. Correct theme to a `web-tag` element
3. Fix code languages for client & server examples

## Test Plan

Manual.

## Related PRs and Issues

N/A.

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes.

### CODE REVIEW

File: src/routes/products/messaging/+page.svelte [hunk 1]
```diff
@@ -88,24 +89,11 @@ try await messaging.createSubscriber(
...
-    'Hello, world!',       // Push body
-    ['news', 'sport'],       // Topic IDs
-);`
-        },
-        {
-            language: 'ts',
-            platform: 'Deno',
-            content: `import { Messaging, ID } from "https://deno.land/x/appwrite/mod.ts";
-
-const messaging = new Messaging(client);
```
Comment (TorstenDittmann) [inline src/routes/products/messaging/+page.svelte:108]: I think you accidentally removed deno 👍🏻

File: src/routes/products/messaging/+page.svelte [hunk 2]
```diff
@@ -568,14 +549,14 @@ messaging.create_email(
                                 class="u-margin-block-start-16 u-min-width-0"
                                 style="margin-block-end: 94px"
                             >
-                                <MultiCodeContextless data={codeTopic} height={450} />
+                                <MultiCodeContextless data={codeTopic} selected="{Platform.ClientWeb}" height={450} />
```
Comment (TorstenDittmann) [inline src/routes/products/messaging/+page.svelte:552]:
```suggestion
                                <MultiCodeContextless data={codeTopic} selected={Platform.ClientWeb} height={450} />
```

File: src/routes/products/messaging/+page.svelte [hunk 3]
```diff
@@ -568,14 +549,14 @@ messaging.create_email(
...
                             </div>
                         </div>
                         <div class="u-flex-vertical u-gap-8 u-min-width-0">
                             <h3 class="web-label web-u-color-text-primary">Send a message</h3>
                             <p class="web-description">Send a message to all targets on a topic.</p>
                             <div class="u-margin-block-start-16 u-min-width-0">
-                                <MultiCodeContextless data={codeMessage} height={450} />
+                                <MultiCodeContextless data={codeMessage} selected="{Platform.ServerNodeJs}" height={450} />
```
Comment (TorstenDittmann) [inline src/routes/products/messaging/+page.svelte:559]:
```suggestion
                                <MultiCodeContextless data={codeMessage} selected={Platform.ServerNodeJs} height={450} />
```

File: src/routes/products/messaging/+page.svelte [hunk 4]
```diff
@@ -599,7 +580,6 @@ messaging.create_email(
                             class="web-card is-normal"
                             href="/docs/products/auth"
                             style="background: rgba(255, 255, 255, 0.04);"
-                            target="_blank"
                             rel="noopener noreferrer"
```
Comment (TorstenDittmann) [inline src/routes/products/messaging/+page.svelte:583]: can you also remove all the `rel="noopener noreferrer"` from relative links?

File: src/routes/products/messaging/+page.svelte (hunk 1 above)
Comment (ItzNotABug) [reply src/routes/products/messaging/+page.svelte:108]: Naa this was intentional.
The task mentioned to use only `Node.js`, so I removed this. Need to add back?

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**